specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/bastionHost.json

```diff
@@ -404,6 +625,219 @@
         }
       },
       "description": "Response for ListBastionHosts API service call."
+    },
+    "BastionShareableLink": {
+      "properties": {
+        "vm": {
+          "$ref": "#/definitions/VM",
+          "description": "Reference of the virtual machine resource."
+        },
+        "bsl": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The unique Bastion Shareable Link to the virtual machine."
+        },
+        "createdAt": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The time when the link was created."
+        },
+        "message": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Optional field indicating the warning or error message related to the vm in case of partial failure"
+        }
+      },
+      "required": [
+        "vm"
+      ],
+      "description": "Bastion Shareable Link."
+    },
+    "BastionShareableLinkListRequest": {
+      "properties": {
+        "vms": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BastionShareableLink"
+          },
+          "description": "List of VM references."
+        }
+      },
+      "description": "Post request for all the Bastion Shareable Link endpoints."
+    },
+    "BastionShareableLinkListResult": {
+      "properties": {
+        "bsls": {
```

Comment:
this should be 'value'

Reply:
example:

"OperationListResult": {
  "description": "Result of the request to list Resource Provider operations. It contains a list of operations and a URL link to get the next set of results.",
  "properties": {
    "value": {
      "type": "array",
      "items": {
        "$ref": "#/definitions/Operation"
      },
      "description": "List of Resource Provider operations supported by the Resource Provider resource provider."
    },
    "nextLink": {
      "type": "string",
      "description": "URL to get the next set of operation list results if there are any."
    }
  }

---
In reply to: [364408441](https://github.com/Azure/azure-rest-api-specs/pull/7969#discussion_r364408441) [](ancestors = 364408441)

Reply:
Updated